### PR TITLE
Add tickets

### PR DESCRIPTION
**Pull request by khjrtbrg**
Tickets landing page! https://github.com/OSFeels/OSFeels.com/issues/45

I added a Google form for requesting ticket/travel assistance, so I'm especially interested in thoughts on that part!

:purple_heart:

### CODE REVIEW

File: app/views/tickets.erb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<div class="row callouts">
+  <div class="col-md-8 col-md-offset-2 text-center">
+    <h2 class="blue">Buy Tickets</h2>
+    <div class="purple">
+      <p>
+        Tickets are now available for OSFeels 2015, starting at $55 for both days.
+      </p>
+      <p>
+        OSFeels is a kind, inclusive, and welcoming conference.<br />
+        We expect all participants to respect our <a href="/conduct"><strong>code of conduct</strong></a> during the event.
```
Comment (jnf) [inline app/views/tickets.erb]: Thank you for including this here!

File: app/views/tickets.erb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <div class="col-md-4 col-md-offset-4">
+    <img class="confetti" src="img/heart_confetti.png">
+  </div>
+</div>
+<div class="row callouts">
+  <div class="col-md-8 col-md-offset-2 text-center">
+    <h2 class="blue">Need Help Getting to OSFeels?</h2>
+    <div class="purple">
+      <p>
+        We'd love to hear from you! <a href="http://goo.gl/forms/Hw9C95aun2" target="_blank"><strong>Get in touch with us to request assistance.</strong></a>
```
Comment (jnf) [inline app/views/tickets.erb]: I'd like to beef up the language here a little bit. Maybe something like...

> We are prioritizing providing financial assistance where we can to folks who would like to attend OS&F, but cannot purchase a ticket on their own. Priority is given to people in the Seattle area, as we have limited funding. Complete this form to request assistance, and we will be in touch soon.

@strand or @theutahkate: What do yinz think? Can you words better than me?

File: app/views/jumbotron.erb [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <div class="visible-sm visible-xs">
       <div class="row newsletter">
         <div class="col-md-8 col-md-offset-2">
-          <%= erb :mailing_list %>
+          <%= erb :tickets_cta %>
```
Comment (kerrizor) [inline app/views/jumbotron.erb]: Should we keep the mailing list or has it served its purpose and outlived its usefulness?

File: app/views/mailing_list.erb [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <div class="hidden-sm hidden-xs">
     <h2>
       <label class="purple" for="mce-EMAIL">
-        Sign up to learn when tickets are available for Open Source &amp;&nbsp;Feelings.
+        Sign up for the latest news from <br> Open Source &amp;&nbsp;Feelings.
```
Comment (kerrizor) [inline app/views/mailing_list.erb]: :+1:

File: app/views/tickets.erb [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+<div class="row callouts newsletter">
+  <div class="col-md-8 col-md-offset-2 text-center">
+    <h2 class="purple">Talk Proposer Discount</h2>
+    <div class="purple">
+      <p>
+        Thanks to everyone who submitted talks!
+      </p>
+      <p>
+        If you proposed a talk, we'll send you a 50% discount code for your ticket shortly.
+      </p>
```
Comment (kerrizor) [inline app/views/tickets.erb]: I don't think we need to include this; let's send it out as part of our "send out the rejection letters" later this week (timing?)

**Pull request merged**